edgegrid-signer-apache-http-client5/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>edgegrid-signer-parent</artifactId>
+        <groupId>com.akamai.edgegrid</groupId>
+        <version>5.0.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>edgegrid-signer-apache-http-client5</artifactId>
+    <packaging>jar</packaging>
+    <name>Apache HTTP Client Library binding for EdgeGrid Client</name>
```

Comment:
`<name>Apache HTTP Client 5 Library binding for EdgeGrid Client</name>`
Otherwise, there are two modules that have the same name and it's slightly confusing (especially in build reports)